manager/controllers/advisories.go

```diff
@@ -202,7 +202,7 @@ func AdvisoriesListIDsHandler(c *gin.Context) {
 	}
 
 	ids := advisoriesIDs(aids)
-	var resp = IDsResponse{IDs: ids}
+	var resp = ids2IDsPlainResponse(ids)
```

Comment:
let's change the return type of `advisoriesIDs` function to return IDsPlainResponse instead of iterating through the list again in `ids2IDsPlainResponse`.

Reply:
Fixed